lib/system/zephyr/poll.c

```diff
@@ -1,22 +1,23 @@
 /*
- * Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+ * Copyright (c) 2022 Qualcomm Innovation Center, Inc.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
 /*
- * @file	zephyr/riscv/sys.c
- * @brief	machine specific system primitives implementation.
+ * @file	zephyr/poll.c
```

Comment:
Could you keep sys.c to  harmonize the file naming with other platforms.

Reply:
done.  force-pushed changes to rename poll.c to sys.c and update CMakeLists.txt accordingly.

(I can't believe that in 2022 github still can't track changes across force-pushes - other tools like gitlab have been doing this for years)

Reply:
The delta is visible by clicking on the compare button or the force_push ;-)

Reply:
oh, thanks!  somehow I expected to see this under Files changed but it appears it's only visible in the PR "conversation".  that's what I get for hanging out in gitlab for a while!